package.json

```diff
@@ -77,6 +77,7 @@
 		"@aws-sdk/client-s3": "^3.554.0",
 		"@aws-sdk/s3-presigned-post": "^3.554.0",
 		"ajv": "^8.12.0",
+		"ajv-formats": "^3.0.1",
```

Comment:
We've tended to lean towards NOT having separate commits just to add packages with no context -- so just smoosh these two commits together (the thinking is that the context for why you're adding a package is the code that uses it, so just group it there!)